uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/addon/EventStorage.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.api.addon;
+
+import lombok.NonNull;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Interface to storage of the events.
+ */
+public interface EventStorage {
```

Comment:
this does not need to be an interface. This can be concrete.

Reply:
Interface removed in 514910c3347e6c98ebf847a9ef2be1514e3aa1a5